### ISSUE TITLE
Decimal Mask: Adding Plus or Minus sign results in an extra groupSeparator

### ISSUE THREAD

**Issue by YoussefTaghlabi** [RobinHerbots/Inputmask#446]
Using the following configuration in v2.5.2:

```
$(<SELECTOR>).inputmask("decimal", { 
                radixPoint: ".", 
                groupSeparator: ",",
                groupSize: 3,
                digits: 2,
                autoGroup: true,
                allowPlus: true,
                allowMinus: true
            });
```
- Type in the following data into the input field: `123456`
  ( This gets masked to: `123,456` )
- Place cursor on the far left. The cursor should be now right before `1`
- Press the minus sign `-`

Expected Result: `-123,456`
Actual Result: `-,123,456`

Please note that the same behavior happens when adding a plus sign `+`.

**Comment by RobinHerbots** [comment]
fixed in version 2.5.4

**Comment by YoussefTaghlabi** [comment]
Thanks
